mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go

```diff
@@ -106,6 +106,35 @@ See [Enabling CMEK for Logging Buckets](https://cloud.google.com/logging/docs/ro
 			},
 		},
 	},
+	"index_configs": &schema.Schema{
```

Comment:
This file needs `gofmt` run on it! `gofmt -w -s mmv1/third_party/terraform/services/logging/resource_logging_project_bucket_config.go` from the repo root should do it.